src/Sylius/Bundle/GridBundle/DependencyInjection/Configuration.php

```diff
@@ -29,12 +29,41 @@ public function getConfigTreeBuilder()
         $treeBuilder = new TreeBuilder();
         $rootNode = $treeBuilder->root('sylius_grid');
 
+        $this->addEnabledDriversSection($rootNode);
         $this->addTemplatesSection($rootNode);
         $this->addGridsSection($rootNode);
 
         return $treeBuilder;
     }
 
+    private function addEnabledDriversSection(ArrayNodeDefinition $node)
+    {
+        // determine which drivers are distributed with this bundle
+        $driverDir = __DIR__ . '/../Resources/config/driver';
+        $iterator = new \RecursiveDirectoryIterator($driverDir);
+        foreach (new \RecursiveIteratorIterator($iterator) as $file) {
+            if ($file->getExtension() !== 'xml') {
+                continue;
+            }
+
+            $validDrivers[] = substr($file->getPathname(), 1 + strlen($driverDir), -4);
+        }
+
+        $node
+            ->children()
+                ->arrayNode('enabled_drivers')
```

Comment:
What do you think about renaming it to `drivers` instead? 

IMO this:
```yaml
sylius_grid:
  drivers:
    - doctrine/orm
```

looks better than this:

```yaml
sylius_grid:
  enabled_drivers:
    - doctrine/orm
```
And it is understandable easily as well